For backend devs: friction getting local build to pass

# [Motivation]
There are dependencies in the project that are causing CI/CD errors.
When fixing these locally, the libraries don't work because python venv isn't fully configured for development mode.

# Description
github CI/CD is failing on mypy and other dependencies.
(1) as a developer, README should help me get my working environment configured
(2) python env setup could install all dependencies required to test/lint/format repo

### Error 1 - Setting up dev environment
to push a PR into repo, user needs to install `mypy` in venv
```
 pip install mypy
```

### Error 2 - Setting up dev environment
mypy throws errors w/ `requests` and [import-untyped]:
```
pdr_backend/util/subgraph.py:10: error: Library stubs not installed for "requests"  [import-untyped]
pdr_backend/util/test/test_subgraph.py:5: error: Library stubs not installed for "requests"  [import-untyped]
```

to fix this, you need to install `types-requests`
```pip install types-requests```

### DoD

- [ ] Issue is fixed

(obsolete list):
- Developer environment and README.md helps user get all packages and dependencies install
- Developer is able to format/lint/typesafe code to push clean PRs relative to CI/CD requirements

--- a/READMEs/backend-dev.md
+++ b/READMEs/backend-dev.md
@@ -9,7 +9,7 @@ This is for core devs to improve pdr-backend repo itself.
 
 ## Install pdr-backend
 
-Follow directions in [predictoor.md](predictoor.md)
+Follow directions to install pdr-backend in [predictoor.md](predictoor.md)
 
 ## Local Network
 

--- a/setup.py
+++ b/setup.py
@@ -16,6 +16,7 @@ install_requirements = [
     "eth-account",
     "eth-keys",
     "matplotlib",
+    "mypy",
     "numpy",
     "pandas",
     "pathlib",
@@ -24,6 +25,7 @@ install_requirements = [
     "pytest-env",
     "requests",
     "scikit-learn",
+    "types-requests",
     "web3",
     "sapphire.py",
     "ocean-contracts==2.0.0a14",  # install this last

